04.ls/ls.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def main
+  files = Dir.glob('*').sort_by(&:downcase)
+  column_count = 3
+  row_count = (files.count.to_f / column_count).ceil
+  columns = split_into_columns(files, column_count, row_count)
+  output(columns, row_count)
+end
+
+def split_into_columns(files, column_count, row_count)
+  column_group = Array.new(column_count) { [] }
+  files.each_with_index do |file, index|
+    col_order = index / row_count
```

Comment:
```suggestion
    col_index = index / row_count
```

配列のindexを指す数値では？と思いました

Reply:
indexとorderを少し混同してしまいました🙇‍♀️確かにorder(順序)ではなく、index(位置を指す)という表現が正しいので、修正いたしました。